File: gluecodium/src/test/resources/smoke/field_constructors/output/android/com/example/smoke/FieldConstructorsInternal.java
/*
 *
 */
package com.example.smoke;
import android.support.annotation.NonNull;
public final class FieldConstructorsInternal {
    @NonNull
    public String publicField;
    public double internalField;
    FieldConstructorsInternal() {
        this.publicField = "foo";
        this.internalField = 42;
    }
    FieldConstructorsInternal(@NonNull final String publicField) {
        this.publicField = publicField;
        this.internalField = 42;
    }
    FieldConstructorsInternal(final double internalField) {
        this.internalField = internalField;
        this.publicField = "foo";
    }
    FieldConstructorsInternal(final double internalField, @NonNull final String publicField) {
        this.internalField = internalField;
        this.publicField = publicField;
    }
}

File: gluecodium/src/test/resources/smoke/field_constructors/output/android-kotlin/com/example/smoke/FieldConstructorsInternal.kt
/*

 *
 */

@file:JvmName("FieldConstructorsInternalExtensions")

package com.example.smoke


class FieldConstructorsInternal {
    @JvmField var publicField: String
    @JvmField var internalField: Double





    internal constructor() {
        this.publicField = "foo"
        this.internalField = 42
    }


    internal constructor(publicField: String) {
        this.publicField = publicField
        this.internalField = 42
    }


    internal constructor(internalField: Double) {
        this.internalField = internalField
        this.publicField = "foo"
    }


    internal constructor(internalField: Double, publicField: String) {
        this.internalField = internalField
        this.publicField = publicField
    }





}

